src/lib/infusion/src/components/inlineEdit/css/InlineEdit.css

```diff
@@ -0,0 +1,64 @@
+/* Themed Appearance + Themed States */
```

Comment:
Can we get this dependence mediated by npm + an install script rather than committing an entire Infusion plus some copy-pasted files?